tests/intmap-properties.hs

```diff
@@ -922,6 +976,52 @@ prop_alter t k = case lookup k t of
     f Nothing   = Just ()
     f (Just ()) = Nothing
 
+#if MIN_VERSION_base(4,8,0)
+-- Verify that the rewrite rules for Identity give the same result as the
+-- non-rewritten version. We use our own TestIdentity functor to compare
+-- against.
+
+data TestIdentity a = TestIdentity { runTestIdentity :: a }
```

Comment:
Use `newtype` instead of `data` to keep things as similar as we can.

Reply:
Done.